apps/remix-ide/src/app/plugins/solcoderAI.tsx

```diff
@@ -47,6 +47,12 @@ export class SolCoder extends Plugin {
     this.solgpt_chat_history = []
   }
 
+  pushChatHistory(prompt, result){
+    const chat:ChatEntry = [prompt, result.data[0]]
+    this.solgpt_chat_history.push(chat)
+    if (this.solgpt_chat_history.length >this.max_history){this.solgpt_chat_history.shift()}
```

Comment:
there should be space after `>`, before `{` and wherever required to make it better readable